docs/application/dotnet/tutorials/certificates/creating-certificates.md

```diff
@@ -0,0 +1,116 @@
+
+![Author certificate is complete](media/certification_guide12.png)
+
+The author certificate should be secure. So we recommend making a backup. You can specify the backup location by clicking `Browse` button.
+
+Be careful because if you lose the author certificate you will not be able to upgrade your application signed with the same certificate.
+
+The `Next` step is creating distributor certificate.
+
+## Create a new distributor certificate
+
+### 1. Create new or select existing distributor certiicate
+
+Select `Create a new distributor certificate`. In case of `Select an existing distributor certificate`, see the [Managing Certificate Profile](managing-certificate-profile.md).
+
+![Distributor certificate](media/certification_guide13.png)
+
+### 2. Information of the distributor certicate
+
+Select the `Privilege` level for the distributor certificate. To ensure that implemented APIs work in the device, the application should be signed with distributor certificate which has appropriate privilege level.
+
+![Set the Privilege and DUID](media/certification_guide15.png)
+
+> **Privileges**  
+> To effectively protect the device system and user private data, the Tizen security architecture is based on privileges and application signing of the Linux basic security model, which includes process isolation and mandatory access control. The users must be able to grant privileges for security-sensitive operations  
+> **Public** - No limitation to use for all Tizen application developers.  
+> **Partner** - No limitation to request and get the distributor certificate with this level. But when the application is uploaded to the Store, Store re-signs the application with official partner level distributor certificate, only for the registered partner developer at Samsung.
+
+ * If you have checked an option to apply the password to be the same as the author certificate to the distributor certificate at [Information of the author certicate](#2-information-of-the-author-certicate), the password is already filled in.If you want to change it, just type the new password for the distributor certificate.
+ * Once the device is connected to the Host PC, the DUID is automatically added to the list. If you want to add more devices, just connect your device to your Host PC via USB or WiFi. Previous DUIDs on the list is not deleted even if you disconnect the device. To delete it click the trash icon. For more information, see [How to connect the device to PC via WiFi](../test/testing-your-app-on-gear.md).
+ * When you need to register devices that are more than 10, upload a txt file using `Import a DUID list file` menu. 1 DUID on a line and max 50 DUIDs are be available.
+
+> The DUID of recent mobile and wearable devices start with a 2.0# but old devices are 1.0#. If a distributor certificate contains a 1.0# DUID, all devices with that certificate need one more step, see the [Permit Device To Install Applications](permit-device-to-install-apps.md) guide.
```

Comment:
Is it a note tag?

Suggestion:
The DUID of recent mobile and wearable devices start with 2.0#,  but old devices are 1.0#. If a distributor certificate contains a 1.0# DUID, all devices with that certificate requires one more step. For more information, see [Permit Device To Install Applications](permit-device-to-install-apps.md).

Reply:
done